src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -643,6 +644,22 @@ protected boolean clickImp(String place) {
         return result;
     }
 
+    @WaitUntil
+    public boolean doubleClick(final String place) {
+        boolean result = false;
+        try {
+            WebElement element = getElementToClick(place);
+            result = doubleClickElement(element);
+        } catch (WebDriverException e) {
```

Comment:
I believe this whole catch block can be removed. The standard `WaitUntil` handling will be sufficient I believe.